src/content/news/2025-04-04-maplibre-newsletter-march-2025/index.mdx

```diff
@@ -0,0 +1,69 @@
+---
+title: "MapLibre Newsletter March 2025"
+date: "2025-04-04"
+categories: ["newsletter"]
+authors: [bart, harel, ramyaragupathy]
+draft: false
+---
+
+import { Image } from "astro:assets";
+import hudScreenshot from "./hud-screenshot.png";
+
+March has been a busy month across the MapLibre ecosystem, with new releases, major community contributions, and powerful features in the works. We’re also incredibly grateful that [MapTiler has joined as a Silver Sponsor of the MapLibre project](https://maplibre.org/news/2025-03-31-maptiler-announcement/). MapTiler has been a long-time supporter of open-source mapping, and their sponsorship helps ensure the sustainability and growth of our community.
+
+With that note, let’s dive into this month’s highlights.
+
+## MapLibre Native
+
+- [Grab](https://en.wikipedia.org/wiki/Grab_Holdings) is [contributing](https://github.com/maplibre/maplibre-native/pulls?q=author%3Agrab-liyanjin) to MapLibre Native. While they currently use an internal fork, they plan to adopt MapLibre Native later this year. We’re looking forward to collaborating with them and supporting their onboarding process.
+
+- MapLibre iOS [6.12.1](https://github.com/maplibre/maplibre-native/releases/tag/ios-v6.12.1) and MapLibre Android [11.8.3](https://github.com/maplibre/maplibre-native/releases/tag/android-v11.8.3), [11.8.4](https://github.com/maplibre/maplibre-native/releases/tag/android-v11.8.4) and [11.8.5](https://github.com/maplibre/maplibre-native/releases/tag/android-v11.8.5) were released. We’ve also started publishing the core as a [static library](https://github.com/maplibre/maplibre-native/releases?q=core-&expanded=true) to support the [Rust wrapper](https://crates.io/crates/maplibre_native) currently in development—note that there are no API stability guarantees at this stage. If you have a use case for this, [let us know](https://github.com/maplibre/maplibre-native/discussions/categories/show-and-tell) about it.
```

Comment:
Development-note? Is this a technical term or an auto correct?

Reply:
I think that should be an em-dash.